app/models/spree/variant.rb

```diff
@@ -221,6 +221,10 @@ def save_default_price
       default_price.save if default_price && (default_price.changed? || default_price.new_record?)
     end
 
+    def find_or_build_default_price
+      default_price || build_default_price
+    end
```

Comment:
I'm curious why this is  needed now, when it wasn't before.
Hmm.. i guess this is what DelegateBelongsTo was automatically doing in `delegator_for`..

Reply:
Yeah, there was a build if not available built into `delegate_belongs_to`. [Here is a little more information](https://github.com/spree/spree/pull/6667/commits/7e1ab357738e74fde5ba153c1ad5603fe5dd79be) if you are interested.

Reply:
Ah ha, thank you.